pages/toolkits/search/google_news.mdx

```diff
@@ -0,0 +1,118 @@
+# Search
+
+import ToolInfo from "@/components/ToolInfo";
+import Badges from "@/components/Badges";
+import TabbedCodeBlock from "@/components/TabbedCodeBlock";
+import TableOfContents from "@/components/TableOfContents";
+
+<ToolInfo
+  description="Enable agents to search for news stories with Google News."
+  author="Arcade"
+  codeLink="https://github.com/ArcadeAI/arcade-ai/tree/main/toolkits/search"
+  authType="API Key"
+  versions={["0.1.0"]}
+/>
+
+<Badges repo="arcadeai/arcade_search" />
+
+The Arcade Google News toolkit provides a pre-built set of tools for interacting with Google News. These tools make it easy to build agents and AI apps that can:
+
+- Search for news stories with Google News.
+
+## Install
+
+```bash
+pip install arcade_search
+```
+
+<Note>
+  pip installing the toolkit is only needed if you are [self-hosting](/home/install/overview) Arcade. You do not need to install the toolkit if you're using Arcade Cloud.
+</Note>
+
+## Default parameters
```

Comment:
consider placing below tools and Auth section. Seems unlikely that someone wants to know how to set a default parameter before seeing the tools